### DISCUSSION TITLE
Dúvida sobre o projeto

### DISCUSSION THREAD

**Discussion by EmmanoelCoutinho** [Nexo-Tec/time#4] [Geral]
Boa tarde gostaria de saber se posso enviar um projeto que ja tenha realizado ou devo começar um novo?

**Comment by ibrahimcesar** [comment]
Primeiro, obrigado pelo interesse @EmmanoelCoutinho ! Você pode fazer uma coisa ou outra, ou ambas – o que estamos buscando é como nos comunicar, colaborar e avaliar remotamente. Então qualquer trabalho seu será muito válido! Fique à vontade em sua submissão! Não precisa ser nada de nossa lista.

**Reply by EmmanoelCoutinho** [reply]
okay, muito obrigado estou terminando os preparativos para o pr!
